### ISSUE TITLE
GET request returning all columns but one from my Postgres DB

### ISSUE THREAD

**Issue by bmwertman** [balderdashy/sails#3135]
I'm using $sailsSocket to make my GET and POST requests. My POST request looks like this;
```javascript
$sailsSocket.post('/schedules/toDateObj', 
    { jobSchedObj: 
        { repair_shop_equipment_id: response.config.data.schedEquip.rsrcId, 
          repair_history_id: response.data.id, 
          technician_id: response.data.technician_id, 
          allotted_time: response.data.allotted_time,
          times: timeObj,
          repair_shop_id: $localstorage.get('shopId')
          }
      })
      .success(function(){
          $location.path('/app/checkin');
      })
      .error(function (response){
          console.log(response);
      });
```
And here is the table it saves to...
![screen shot 2015-08-01 at 2 17 15 pm](https://cloud.githubusercontent.com/assets/2573903/9065614/b73bf1ea-3aa0-11e5-9ff5-75febb50ef77.png)

Everything saves fine I even threw a console.log in my ScheduleController to check what was being inserted in the database with my POST request. Here is the output;
![screen shot 2015-08-03 at 4 41 09 pm](https://cloud.githubusercontent.com/assets/2573903/9067464/9242e0fa-3aab-11e5-84dc-2f5f3956dd71.png)

The problem is when I make a GET request to the `Schedule` table it comes back with everything except `repair_shop_equipment_id`. Here is the get request;

```javascript
 var firstDay = scope.getJobsFor[0];
                      var lowerBound = firstDay.open_time;
                      var lastDay = scope.getJobsFor[scope.getJobsFor.length - 1];
                      var upperBound = lastDay.close_time;
                      $sailsSocket.get("/schedules", {params:
                        {where: {
                          repair_shop_id: scope.shopId,
                          technician_id: scope.schedTech.id,  
                          repair_shop_equipment_id: scope.schedEquip.rsrcId,
                          scheduled_start_time:{ date: {'>':lowerBound, '<':upperBound}}
                          }
                        }
                      })
                      .success(function (response){
```

And this is the response...
![screen shot 2015-08-04 at 1 24 25 pm](https://cloud.githubusercontent.com/assets/2573903/9067572/3cd16fdc-3aac-11e5-8673-03a668bbc67d.png)

Here is a link to a Gist with my associated Sails models and the schedule controller;
https://gist.github.com/bmwertman/c9a426674a6e4767cd35

Is there something wrong with one of my model configurations? Thanks.

**Comment by bmwertman** [comment]
I've found a way to retrieve the `repair_shop_equipment_id` with the GET request to '/schedules'. Although it makes zero sense as to why it actually works. I'll explain why;

if I change my GET request as follows it would make at least a bit more sense that `repair_shop_equipment_id` gets returned with the response;

```javascript
 var firstDay = scope.getJobsFor[0];
                      var lowerBound = firstDay.open_time;
                      var lastDay = scope.getJobsFor[scope.getJobsFor.length - 1];
                      var upperBound = lastDay.close_time;
                      $sailsSocket.get("/schedules/?populate=repair_shop_equipment_id", {params:
                        {where: {
                          repair_shop_id: scope.shopId,
                          technician_id: scope.schedTech.id,  
                          repair_shop_equipment_id: scope.schedEquip.rsrcId,
                          scheduled_start_time:{ date: {'>':lowerBound, '<':upperBound}}
                          }
                        }
                      })
                      .success(function (response){
```

But when I do this I get this...
![screen shot 2015-08-06 at 2 13 59 pm](https://cloud.githubusercontent.com/assets/2573903/9121965/eda83df6-3c53-11e5-8f4c-d2af7cf5ebea.png)
...Notice the `technician_id` changed from an object with property `id` to simply the value of `id` and the `repair_shop_equipment_id` is still not included.

 If I make the request like this though;
```javascript
 var firstDay = scope.getJobsFor[0];
                      var lowerBound = firstDay.open_time;
                      var lastDay = scope.getJobsFor[scope.getJobsFor.length - 1];
                      var upperBound = lastDay.close_time;
                      $sailsSocket.get("/schedules/?populate=technician_id", {params:
                        {where: {
                          repair_shop_id: scope.shopId,
                          technician_id: scope.schedTech.id,  
                          repair_shop_equipment_id: scope.schedEquip.rsrcId,
                          scheduled_start_time:{ date: {'>':lowerBound, '<':upperBound}}
                          }
                        }
                      })
                      .success(function (response){
```
...Notice I'm now asking it to populate with the `technician_id`

I get back what I wanted all along. And it looks like this;
![screen shot 2015-08-06 at 4 02 05 pm](https://cloud.githubusercontent.com/assets/2573903/9122124/cd442a38-3c54-11e5-85c7-65d2e2a85287.png)
...See `repair_shop_equipment _id` is in the response.

But why did I have to ask it to populate `technician_id` to get `repair_shop_equipment_id` to come back with the response? Better yet why do I have to pass anything additional with the request to get the entire table row back?

**Comment by sailsbot** [comment]
Thanks for posting, @bmwertman.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
